### PR TITLE
DOC-5355: Updated Metrics page

### PR DESCRIPTION
**Pull request by AnnaK-R3**

### CODE REVIEW

**Review by relyafi — COMMENTED** [review]
Couple of minor suggestions re. formatting:

- I think we should review column widths. There's a lot of splitting across multiple lines for Metric, Type and Tag due to very small column sizes being used in some instances. Widths are also inconsistent across the categories.
- It might be cleaner to use bulleted lists for the tags?

**Review by yarivus — APPROVED** [review]
lgtm

**Comment by AnnaK-R3** [comment]
> Couple of minor suggestions re. formatting:
> 
> * I think we should review column widths. There's a lot of splitting across multiple lines for Metric, Type and Tag due to very small column sizes being used in some instances. Widths are also inconsistent across the categories.
> * It might be cleaner to use bulleted lists for the tags?

I applied your formatting suggestions @relyafi and the tables look much better. Thanks for the review!

**Review by relyafi — DISMISSED** [review]
LGTM

**Review by yarivus — APPROVED** [review]
LGTM

File: content/en/platform/corda/5.0/deploying-operating/observability/metrics.md
Comment (davidcurrie) [inline content/en/platform/corda/5.0/deploying-operating/observability/metrics.md:54]:
```suggestion
* Sandbox
```

**Review by yarivus — DISMISSED** [review]
LGTM

**Review by yarivus — DISMISSED** [review]
LGTM

**Review by yarivus — APPROVED** [review]
LGTM

**Review by yarivus — APPROVED** [review]
LGTM

**Review by yarivus — DISMISSED** [review]
LGTM

File: content/en/platform/corda/5.0/deploying-operating/observability/metrics.md
Comment (paulmoloneyr3) [inline content/en/platform/corda/5.0/deploying-operating/observability/metrics.md:76]:
```suggestion
    width: 25%;
```

I'd probably make this 35% or so so the first column doesn't need to break. Additionally, could narrow the second column and then make the third one wider.

File: content/en/platform/corda/5.0/deploying-operating/observability/metrics.md
Comment (paulmoloneyr3) [inline content/en/platform/corda/5.0/deploying-operating/observability/metrics.md:92]:
```suggestion
| `corda_http_server_request_time_seconds` | Timer | `address` | HTTP requests processing time; the `address` tag is the address to which the metric is applicable. |
```

File: content/en/platform/corda/5.0/deploying-operating/observability/metrics.md
Comment (paulmoloneyr3) [inline content/en/platform/corda/5.0/deploying-operating/observability/metrics.md:97]:
```suggestion
configuration, allowing a safe execution environment within a JVM process that provides isolation for CorDapps.
```

File: content/en/platform/corda/5.0/deploying-operating/observability/metrics.md
Comment (paulmoloneyr3) [inline content/en/platform/corda/5.0/deploying-operating/observability/metrics.md:125]:
```suggestion
| `corda_flow_pipeline_execution_time_seconds` | Timer | <ul><li>`flow.class`</li></ul> | The total time that a flow spent processing in the pipeline, rather than queued (includes fiber execution time.) |
```

File: content/en/platform/corda/5.0/deploying-operating/observability/metrics.md
Comment (paulmoloneyr3) [inline content/en/platform/corda/5.0/deploying-operating/observability/metrics.md:126]:
```suggestion
| `corda_flow_fiber_execution_time_seconds` | Timer | <ul><li>`flow.class`</li></ul> | The total time a flow spent executing user code in the fiber. |
```

**Review by paulmoloneyr3 — CHANGES_REQUESTED** [review]

File: content/en/platform/corda/5.0/deploying-operating/observability/metrics.md
Comment (AnnaK-R3) [reply content/en/platform/corda/5.0/deploying-operating/observability/metrics.md:91]: leaving as is

**Review by yarivus — COMMENTED** [review]
LGTM - pending comments

File: content/en/platform/corda/5.0/deploying-operating/observability/metrics.md
Comment (AnnaK-R3) [reply content/en/platform/corda/5.0/deploying-operating/observability/metrics.md:76]: Reformatted the table

**Review by paulmoloneyr3 — APPROVED** [review]

**Review by paulmoloneyr3 — APPROVED** [review]

File: content/en/platform/corda/5.0/deploying-operating/observability/metrics.md
Comment (AnnaK-R3) [reply content/en/platform/corda/5.0/deploying-operating/observability/metrics.md:105]: It look to me that this is a generic term used in IT. Leaving as is.

**Pull request merged**
